tensorflow/compiler/xla/service/hlo_rematerialization.cc

```diff
@@ -418,11 +419,16 @@ class MemoryUsageTracker {
   // the given uses.
   Buffer& RematerializeBuffer(const Buffer& original_buffer, Item* remat_item,
                               ItemList&& rematerialized_uses) {
-    CHECK(original_buffer.defining_instruction->placed);
-    CHECK(!original_buffer.has_indirect_uses);
-    CHECK(!original_buffer.live_out);
+    CHECK(original_buffer.defining_instruction->placed)
+    "Instruction " << original_buffer.defining_instruction->instruction->name()
```

Comment:
This is missing a << at the beginning (weird that it compiles at all?)

Reply:
Thanks for review, probably was taking as two different statements. Fixed.

Reply:
Thanks for review, probaby was taking as a two different statements. Fixed.